.ci/Jenkinsfile

```
@@ -118,6 +118,19 @@ pipeline {
         }
       }
     }
+    stage('Deploy - Package Registry') {
+      options { skipDefaultCheckout() }
+      when { expression { isTag() } }
+      steps {
+        build(job: "deploy-package-registry",
```

Nit. Nothing is deployed here, but released, or tagged.
```suggestion
        build(job: "release-package-registry-distribution",
```